intelmq/bots/BOTS

```diff
@@ -728,6 +728,18 @@
                 "port": "<port>"
             }
         },
+        "Redis": {
+            "description": "Redis is the bot responsible to send events to a Redis server.",
+            "module": "intelmq.bots.outputs.redis.output",
+            "parameters": {
+                "redis_server_ip": "127.0.0.1",
+                "redis_server_port": "6379",
+                "redis_db": "2",
+                "redis_queue": "external-redis-queue",
+                "redis_timeout": "50000",
```

Comment:
this BOTS file makes me crazy...
What is wrong here?
        },
        "PostgreSQL": {
            "description": "PostgreSQL is the bot responsible to send events to a PostgreSQL Database. When activating autocommit, transactions are not used: http://initd.org/psycopg/docs/connection.html#connection.autocommit",
            "module": "intelmq.bots.outputs.postgresql.output",
            "parameters": {
                "autocommit": true,
                "database": "intelmq-events",
                "host": "localhost",
                "password": "<password>",
                "port": "5432",
                "sslmode": "require",
                "table": "events",
                "user": "intelmq"
            }
         },
         "Redis": {
               "description": "Redis is the bot responsible to send events to a Redis server.",
               "module": "intelmq.bots.outputs.redis.output",
               "parameters": {
                   "redis_db": "2",
                   "redis_password": "<password>",
                   "redis_queue": "external-redis-queue",
                   "redis_server_ip": "127.0.0.1",
                   "redis_server_port": "6379",
                   "redis_timeout": "50000"
               }
        },
        "REST API": {
            "description": "REST API is the bot responsible to send events to a REST API listener through POST.",
            "module": "intelmq.bots.outputs.restapi.output",
            "parameters": {
                "auth_token": "<token>",
                "auth_token_name": "<token name>",
                "host": "<host>"
            }
        },
        "TCP": {


why this?

python3 test_conf.py 
F......
======================================================================
FAIL: test_BOTS_syntax (__main__.TestConf)
Test if BOTS has correct syntax.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_conf.py", line 75, in test_BOTS_syntax
    self.assertEqual(to_json(interpreted), fcontent)
AssertionError: '{\n [40114 chars]     },\n        "REST API": {\n            "d[18044 chars]n}\n' != '{\n [40114 chars]      },\n         "Redis": {\n               [18076 chars]n}\n'
Diff is 59858 characters long. Set self.maxDiff to None to see it.

----------------------------------------------------------------------
Ran 7 tests in 0.017s

FAILED (failures=1)